pavelib/assets.py

```diff
@@ -767,10 +768,23 @@ def webpack(options):
     static_root_lms = Env.get_django_setting("STATIC_ROOT", "lms", settings=settings)
     static_root_cms = Env.get_django_setting("STATIC_ROOT", "cms", settings=settings)
     config_path = Env.get_django_setting("WEBPACK_CONFIG_PATH", "lms", settings=settings)
-    environment = u'NODE_ENV={node_env} STATIC_ROOT_LMS={static_root_lms} STATIC_ROOT_CMS={static_root_cms}'.format(
+    additional_node_env_vars_sorted_json = json.dumps(
+        json.loads(
+            Env.get_django_setting(  # pylint: disable=no-member
+                "ADDITIONAL_NODE_ENV_VARS",
+                "cms",
+                settings=settings
+            ).replace("'", '"')
+        ),
+        sort_keys=True,
+    )
+    additional_node_env_vars = json.dumps(additional_node_env_vars_sorted_json)
```

Comment:
`json.dumps` is already called in the previous step while defining the variable.

Reply:
Check [this commit please](https://github.com/open-craft/edx-platform/pull/271/commits/63581182bef5bc641f5f8a5a1973e753b67334fc).

So in that commit, what I did was take the `Env.get_django_setting("ADDITIONAL_NODE_ENV_VARS", "cms",
     settings=settings).replace("'", '"')` and simply sort it. So I `json.loads` to sort it and then `json.dumps` after it is sorted.

But we need to run `json.dumps` again in order to escape the previous string, which is the dump of the sorted json.

So we dump twice in order to make sure that the string is escaped when it is added to the command.